csp/contrib/rate_limiting.py

```diff
@@ -16,9 +16,32 @@ def build_policy(self, request, response):
         replace = getattr(response, "_csp_replace", {})
         nonce = getattr(request, "_csp_nonce", None)
 
-        report_percentage = getattr(settings, "CSP_REPORT_PERCENTAGE")
-        include_report_uri = random.random() < report_percentage
+        policy = getattr(settings, "CONTENT_SECURITY_POLICY", {})
```

Comment:
It feels like if `CONTENT_SECURITY_POLICY` isn't defined, it should default to `None` and not `{}`.